modular_skyrat/modules/death_consequences_perk/death_consequences_customization.dm

```diff
@@ -189,40 +208,43 @@
 	minimum = 0
 	maximum = 150
 
+	config_name = "Crit threshold reduction: Maximum reduction"
+	config_desc = "When at the ending degradation percent, crit threshold will be reduced by this, \
+	with lower percentages causing equally displaced reducions, such as having 50% degradation causing 50% of this to be applied."
+
 /datum/preference/numeric/death_consequences/max_stamina_damage/create_default_value()
 	return 80
 
 // BOOLEANS
 
 /datum/preference/toggle/death_consequences
 	abstract_type = /datum/preference/toggle/death_consequences
-	category = PREFERENCE_CATEGORY_NON_CONTEXTUAL
+	category = "Test"
 	savefile_identifier = PREFERENCE_CHARACTER
 
-/datum/preference/toggle/death_consequences/is_accessible(datum/preferences/preferences)
-	. = ..()
-	if (!.)
-		return FALSE
+	/// The name to be shown in the config window.
```

Comment:
Didn't you already add something to do exactly this? Did you just forget to remove this code?